plugins/parquet-store/parquet_store.cpp

```diff
@@ -0,0 +1,722 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2022 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <vast/arrow_table_slice_builder.hpp>
+#include <vast/chunk.hpp>
+#include <vast/concept/convertible/data.hpp>
+#include <vast/concept/convertible/to.hpp>
```

Comment:
You're not using this and some other headers, e.g., base64, zip_iterator, convertible/to, io/write. Can you do a pass to remove unnecessary includes?